src/main/java/me/blackvein/quests/Event.java

```diff
@@ -232,13 +241,51 @@ public void fire(Quester quester, Quest quest) {
 		if (failQuest == true) {
 			quest.failQuest(quester);
 		}
+		if (timer > 0) {
+			player.sendMessage(Quests.parseString(String.format(Lang.get("timerStart"), timer), quest));
+			if (timer > 60) {
+				quester.timers.put(plugin.getServer().getScheduler().scheduleAsyncDelayedTask(plugin, new ObjectiveTimer(plugin, quester, quest, 60, false),	(timer-60)*20), quest);
+			}
+			if (timer > 30) {
+				quester.timers.put(plugin.getServer().getScheduler().scheduleAsyncDelayedTask(plugin, new ObjectiveTimer(plugin, quester, quest, 30, false),	(timer-30)*20), quest);
+			}
+			if (timer > 10) {
+				quester.timers.put(plugin.getServer().getScheduler().scheduleAsyncDelayedTask(plugin, new ObjectiveTimer(plugin, quester, quest, 10, false),	(timer-10)*20), quest);
+			}
+			if (timer > 5) {
+				quester.timers.put(plugin.getServer().getScheduler().scheduleAsyncDelayedTask(plugin, new ObjectiveTimer(plugin, quester, quest, 5, false),	(timer-5)*20), quest);
+			}
+			if (timer > 4) {
+				quester.timers.put(plugin.getServer().getScheduler().scheduleAsyncDelayedTask(plugin, new ObjectiveTimer(plugin, quester, quest, 4, false),	(timer-4)*20), quest);
+			}
+			if (timer > 3) {
+				quester.timers.put(plugin.getServer().getScheduler().scheduleAsyncDelayedTask(plugin, new ObjectiveTimer(plugin, quester, quest, 3, false),	(timer-3)*20), quest);
+			}
+			if (timer > 2) {
+				quester.timers.put(plugin.getServer().getScheduler().scheduleAsyncDelayedTask(plugin, new ObjectiveTimer(plugin, quester, quest, 2, false),	(timer-2)*20), quest);
+			}
+			if (timer > 1) {
+				quester.timers.put(plugin.getServer().getScheduler().scheduleAsyncDelayedTask(plugin, new ObjectiveTimer(plugin, quester, quest, 1, false),	(timer-1)*20), quest);
+			}
+			quester.timers.put(plugin.getServer().getScheduler().scheduleAsyncDelayedTask(plugin, new ObjectiveTimer(plugin, quester, quest, timer - 30, true),	timer*20), quest);
+		}
+		if (cancelTimer) {
+			quester.timers.entrySet().stream()
+					.filter(questEntry -> Objects.equals(questEntry.getValue().getName(), quest.getName()))
```

Comment:
Would like to avoid Java 8 dependency if possible (the "->" lambda expression). This is because Quests supports older versions of the Minecraft server which will not necessarily be run with Java 8.